xonsh/base_shell.py

```diff
@@ -226,7 +226,7 @@ def reset_buffer(self):
 
     def settitle(self):
         """Sets terminal title."""
-        _ = self
+        _ = self  # noqa: F841
```

Comment:
I thought this was put there for pylint... but if you are ok to remove it I will 👍